File: test/random/pipelines/TestExternalUse/code/testexternaluse/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from testexternaluse.config.ConfigStore import *
from testexternaluse.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: testprojects/master_scala_main/pipelines/TestPipeline1/code/src/main/scala/io/prophecy/pipelines/testpipeline/graph/Reformat_1.scala
package io.prophecy.pipelines.testpipeline.graph

import io.prophecy.libs._
import io.prophecy.pipelines.testpipeline.config.ConfigStore._
import io.prophecy.pipelines.testpipeline.udfs.UDFs._
import io.prophecy.pipelines.testpipeline.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.select(concat(col("`c  - int`"), col("`c -  boolean _  `")).as("c1"))

}
